email-sending-concepts/dkim.md

```diff
@@ -0,0 +1,41 @@
+# 📬 What Does DKIM Stand For?
+
+**DKIM** stands for **DomainKeys Identified Mail**.
+
+It is an email authentication method that allows the **sender to digitally sign emails** using a private key. The receiving server can then verify the email’s authenticity using the sender’s **public key published in DNS**.
+
+DKIM helps ensure that the email was not **altered in transit** and truly comes from the claimed domain.
+
+---
+
+## 🔍 How DKIM Works
+
+1. The sender's mail server adds a **DKIM-Signature** header to the email
+2. That header includes a **hash of the message** and metadata, signed using the domain's private key
+3. The recipient's mail server looks up the **public key in DNS** (via a TXT record at `selector._domainkey.example.com`)
+4. It uses the public key to verify the signature and check the integrity of the email
+
+If the signature matches → **DKIM = PASS** ✅
+
+---
+
+## 🧪 Example DKIM DNS Record
+
+```dns
```

Comment:
this throws this warning "The language 'dns' is not loaded, falling back to 'txt' for syntax highlighting."